applications/IgaApplication/custom_modelers/nurbs_geometry_modeler.cpp

```diff
@@ -0,0 +1,193 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Manuel Messmer
+//
+
+// Project includes
+#include "nurbs_geometry_modeler.h"
+#include "includes/define.h"
+
+namespace Kratos
+{
+    ///@name Stages
+    ///@{
+
+    void NurbsGeometryModeler::SetupGeometryModel(){
+        // Get bounding box
+        KRATOS_ERROR_IF_NOT(mParameters.Has("lower_point"))
+            << "NurbsGeometryModeler: Missing \"lower_point\" section" << std::endl;
+        double x_a =  mParameters["lower_point"].GetArrayItem(0).GetDouble();
+        double y_a =  mParameters["lower_point"].GetArrayItem(1).GetDouble();
+        double z_a =  mParameters["lower_point"].GetArrayItem(2).GetDouble();
+        Point point_a(x_a,y_a,z_a);
```

Comment:
```suggestion
        Point point_a(mParameters["lower_point"].GetVector());
```
sth like this might work

Reply:
Does work, indeed!!